### ISSUE TITLE
Multiple box shadows appear with a gap in the top left corner

### ISSUE THREAD

**Issue by osener** [vercel/satori#339]
# Bug report
Given the following div with multiple box shadows, browser rendering and Satori SVG output look different.

[OG Playground link](https://og-playground.vercel.app/?share=ZVLbboMwDP0Vy9KkTUIalLZqs64v035g2yMvoQmQLSWIhNEK8e9zuGxrK-HgYx_OIU46PBghkeFOqG-w7qzlc9eBULbS_MwgwUzLU4IBpPzwldemKcWL0ab2rbZQTvpeq4QrGKzCMIBCqrxwjHLo-31Sglem16-4zwEqY5VTpvQ6PLVGN4PU2Ly1H-uTT0w-Y2Ey-1fRMiO8naEzlf-vCaWmFrJ-40I1lsFf9fRecGFaNmIg27A6wXKKmKLOU35PuxufxerugRKqz-F5AydeBjBHFIbXvM3MW0Qr0onWftleUCO_xOvpg0vjaH0tGN1wiJKg30nfJ-UjncDukU5g_5SUGKCp_NAtsg6HYSLb0OhwnCOypQdCpk2OLOPaygDl0Xyqj3PlL4lrB0Q6mSnd6zGVApmrG9n3Pw)

### Input
```jsx
<div style={{ display: "flex", backgroundColor: "white", width: 500, height:500 }}>
 <div
  style={{
    position: "absolute",
    display: "flex",
    width: 300,
    height: 300,
    left: 90,
    top: 50,
    borderRadius: 0,
    boxShadow:
      "0px 4px 4px 3px rgba(0, 0, 0, 25%), 0px 0px 0px 4px rgba(34, 34, 34, 100%), 0px 0px 0px 8px rgba(215, 216, 219, 100%), 0px 10px 36px 0px rgba(0, 0, 0, 16%), 0px 0px 0px 1px rgba(0, 0, 0, 6%)"
  }}
/>
</div>;
```
### SVG (Satori)
![image](https://user-images.githubusercontent.com/111265/208122492-4e7009ca-06a7-40a6-ab5d-dadd19ab034b.png)

### HTML (Native)
![image](https://user-images.githubusercontent.com/111265/208122580-81e4d1db-de6e-450b-a251-9d3dd5423bf2.png)

**Comment by jeetiss** [comment]
I can reproduce this bug with one `box-shadow` and `borderRadius: 0` and i fix it in #388

**Comment by osener** [comment]
This is great, big thanks for the fix @jeetiss!
